tests/not_test_data/test_transform.py

```diff
@@ -0,0 +1,10 @@
+import torch
+from src.models.conv_nn import ConvNet
+from src.models.kornia_trans import transform
+
+
+def test_transform():
+    data = torch.rand([64, 3, 224, 224]) # random datas
+    model = ConvNet(1028,512)
```

Comment:
I think this line can be removed

Reply:
indeed, flake8 was raising this error. fixed with bb191ef.